packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -338,6 +338,7 @@
 +  enabled: false
 +  metricsPort: 10254
 +  component: ingress-nginx
++  networkPolicyEnabled: false
```

Comment:
This is OK for now - because it seems like it should only need to be on or off. However, other `networkPolicy` related values in this file make it an object. So if we need more than one value for this I would conform to that example as well. I.e.

```yaml
networkPolicy:
  - enabled: false
  - namespace: null # user must set
  ```

Reply:
That was my initial thought when introducing this value. But decided against it as it's a single value.

Reply:
Its a single value, but if we need to make it an object later, that could affect upgrade paths / is not backward compatible. Unless there is a specific reason this needs to be a single value, it would be better as an object.

Good catch @mallardduck